### PR TITLE
chore: enabling kubemonkey on all pods

### PR DESCRIPTION
**Pull request by nicolasburtey**

### CODE REVIEW

File: charts/galoy/values.yaml [hunk 1]
```diff
@@ -65,7 +65,7 @@ galoy:
       repository: us.gcr.io/galoy-org/galoy-consent
       ## Digest of the image
       ##
-      digest: "sha256:be5e90bffe3ae6dc6f2729266b21022802fbed803341a90515c3ed0da6d9def1" # METADATA:: repository=https://github.com/GaloyMoney/galoy;commit_ref=84f91fb;app=consent;monorepo_subdir=apps/consent;
+      digest: "sha256:5d6ebf60a7c64aa42ccb1dd346b170be22ca5d18f02d44b249a848ba3a0ea9e4" # METADATA:: repository=https://github.com/GaloyMoney/galoy;commit_ref=d2f6f35;app=consent;monorepo_subdir=apps/consent;
```
Comment (krtk6160) [inline charts/galoy/values.yaml:68]: What is this change?

File: charts/admin-panel/templates/deployment.yaml [hunk 2]
```diff
@@ -6,6 +6,9 @@ metadata:
     app: {{ template "adminPanel.fullname" . }}
     chart: "{{ .Chart.Name }}-{{ .Chart.Version }}"
     release: "{{ .Release.Name }}"
+{{- with .Values.labels }}
+{{ toYaml . | trim | nindent 6 }}
```
Comment (krtk6160) [inline charts/admin-panel/templates/deployment.yaml:10]: The indent looks wrong. Should be 4 I think.

File: charts/admin-panel/templates/deployment.yaml [hunk 3]
```diff
@@ -17,8 +20,8 @@ spec:
       labels:
         app: {{ template "adminPanel.fullname" . }}
         release: "{{ .Release.Name }}"
-{{- with .Values.labels }}
-{{ toYaml . | trim | indent 8 }}
+{{- with .Values.podLabels }}
+{{- toYaml . | nindent 8 }}
```
Comment (krtk6160) [inline charts/admin-panel/templates/deployment.yaml:24]: Should be 6 here.

File: charts/admin-panel/values.yaml [hunk 4]
```diff
@@ -21,3 +21,12 @@ secrets:
...
 tracingServiceName: "admin-panel"
+labels:
+  - kube-monkey/enabled: enabled
+  - kube-monkey/identifier: admin-panel
+  - kube-monkey/kill-mode: fixed
+  - kube-monkey/kill-value: "1"
+  - kube-monkey/mtbf: "3"
+podLabels:
+  - kube-monkey/enabled: enabled
+  - kube-monkey/identifier: admin-panel
```
Comment (krtk6160) [inline charts/admin-panel/values.yaml:32]: Both of these should be objects, not arrays. You can validate the syntax by using `helm template admin-panel .` in `charts/charts/admin-panel`

Comment (nicolasburtey) [reply charts/admin-panel/values.yaml:32]:
```
% helm template .
Error: template: admin-panel/templates/secrets.yaml:12:54: executing "admin-panel/templates/secrets.yaml" at <b64enc>: invalid value; expected string
```

Comment (krtk6160) [reply charts/admin-panel/values.yaml:32]: Pushed a fix, should work now.

Comment (nicolasburtey) [reply charts/admin-panel/values.yaml:32]: if I commented the value I could get it to work

File: charts/galoy/values.yaml (hunk 1 above)
Comment (nicolasburtey) [reply charts/galoy/values.yaml:68]: not sure why and how it showed up. reverting the change

**Review by krtk6160 — APPROVED** [review]

**Pull request merged**